plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/util/CloudSdkProcessWrapper.java

```diff
@@ -71,26 +70,25 @@ public class CloudSdkProcessWrapper {
   /**
    * Sets up a {@link CloudSdk} to be used for App Engine deploy.
    */
-  public Deployment getAppEngineDeployment(Path credentialFile,
-      MessageConsoleStream normalOutputStream) throws CloudSdkNotFoundException {
-    //Preconditions.checkNotNull(credentialFile, "credential required for deploying");
-    //Preconditions.checkArgument(Files.exists(credentialFile), "non-existing credential file");
+  public Deployment getAppEngineDeployment(MessageConsoleStream normalOutputStream) 
+      throws CloudSdkNotFoundException {
+    Credential credential = null;
+    try {
+      credential = new GoogleApiFactory().getAccount().getOAuth2Credential();
+    }
+    catch (IOException ex) {
+      // will be handled in finally block;
+    }
+    finally {
```

Comment:
Is this `finally` necessary? Consider for removal, and just placing the precondition in line with the one below it.

Reply:
Done